docs/oscar_references.bib

```diff
@@ -33,6 +33,13 @@ @Book{AL94
   url           = {https://doi.org/10.1090/gsm/003}
 }
 
+@Misc{Aut00,
+  author        = {{A}uthors, {T}he {S}tacks {P}roject},
+  title         = {{S}tacks {P}roject},
+  year          = {0000},
+  url           = {https://stacks.math.columbia.edu/}
```

Comment:
My view on this is: this is just not a sensible bib entry. Can't you just put a URL link into the text as we do for e.g. the Singular homepage

Reply:
(but there is no reason to delay merging this PR because of this)